docs/source/glossary.rst

```diff
@@ -143,3 +143,14 @@ positional embeddings.
 
 Absolute positional embeddings are selected in the range ``[0, config.max_position_embeddings - 1]``. Some models
 use other types of positional embeddings, such as sinusoidal position embeddings or relative position embeddings.
+
+
+Feed Forward Chunking
```

Comment:
@LysandreJik @thomwolf - not sure whether glossary is the correct place for this. Since Feed Forward chunking can very easily be applied to all models I thougth it's best to put it here.